test/mocha.opts

```diff
@@ -1,4 +1,4 @@
-test/main.js test/simulator/**/*.js test/application/*.js test/chat-plugins/*.js test/dev-tools/*.js
+test/main.js test/dev-tools/**/*.js test/server/**/*.js test/sim/**/*.js
```

Comment:
I placed dev-tools last tactically so it wouldn't be the first test run and have to pay the `Dex` loading penalty on the test which is already on the slow side. The real answer is to probably disable timeouts for those tests/set them to be long? They're currently `skip`ped though, so it doesn't really matter.

Reply:
Oh, I see. We should preload ``Dex`` in ``main.js``.